### PR TITLE
feat(Web, Home): API Integration

### PR DESCRIPTION
**Pull request by r17x**

### CODE REVIEW

**Comment by faultables** [comment]
The logs is [here](https://privatebin.net/?8f264a4c57f69105#AHS9r1AsLS6nLFrSjeL8XYYG3NDbKKQnnGTH4XMzmfJN) in case you can't access it

File: web/package.json [hunk 1]
```diff
@@ -4,20 +4,29 @@
...
-    "reason-react": ">=0.7.0"
+    "reason-react": ">=0.7.0",
+    "relude": "^0.55.0",
+    "relude-fetch": "^0.6.0",
+    "relude-reason-react": "^0.8.2",
+    "webpack-dev-server": "^3.10.3"
   },
   "devDependencies": {
-    "bs-platform": "^7.0.1",
+    "bs-platform": "^6.2.1",
```
Comment (faultables) [inline web/package.json]: ada apa di downgrade?

**Review by faultables — CHANGES_REQUESTED** [review]
Screen Shot 2020-03-29 at 12 31 08 PM

**Review by faultables — APPROVED** [review]

**Comment by faultables** [comment]
salah pencet tadi hampura

File: web/package.json (hunk 1 above)
Comment (r17x) [reply web/package.json]: waktu ngetrace gan kenapa gak work di alpine. gw balikin lagi yah

Comment (faultables) [reply web/package.json]: bebas gan

**Pull request merged**